docs/features/index.md

```diff
@@ -1,3 +1,69 @@
 # Overview
 
-This section has all the features of Argo Rollouts. Check out [rollout.yaml](rollout.yaml) for a fully annotated Rollout.
\ No newline at end of file
+The Rollout object has two available strategies: Canary and BlueGreen. Below are the links to the documenation for each strategy:
+1. [Blue Green](/feature/bluegreen/)
+1. [Canary](/feature/canary/)
+
+Here is a rollout that has every field automated:
```

Comment:
> Here is a rollout that has every field automated:
 
Do you mean documented?

Maybe just say:

The following describes all the available fields of a rollout: